Production Releasing.md

```diff
@@ -6,26 +6,34 @@ Not all checklist items are necessary in every app.
 
 If your not sure whether you should add some feature (e.g. Google Analytics) to the webapp your releasing, consult with the project manager and/or your team leader.
 
+In your project management app (e.g. Productive, Trello...) create a task _Production Checklist_ and link to this handbook chapter. Finish it prior to production releasing.
+
 ## Checklist items:
 
-* Test the app on various browsers. The usual set of browsers is (this can vary from project to project): Chrome, FireFox, Opera, Safari, IE8+. HTML apps only.
-* Go through a final design review with the designer. HTML apps only.
-* Handout the application to the QA team so they can try to find potential bugs before the client or customers do. HTML apps only.
 * Are you uploading all files to S3 and not to the local (public/system) filesystem?
-* Did you add your application to the [Application list](https://github.com/infinum/internal-guides/blob/master/applications/Readme.md)
-* Did you add monitoring via UptimeRobot?
-* If the project is on code climate, check that it doesn't have any security issues. If the project isn't on code climate, check the security of your application with [Brakeman](https://github.com/presidentbeef/brakeman)
-* Where's the DNS zone? If there are no dependencies (like Cpanel, lots of subdomains etc..), you're probably better off hosting it on our DNS servers (PointHQ) because it's faster to do changes then contacting a third party DNS hoster
-* Lower TTL for all A/CNAME records to 300.
-* Are you using SSL?
-* Add the cookies_eu gem.
+* Check that the app doesn't have any security issues on Code Climate. If it isn't on Code Climate, why isn't it? If there is a explicit reason why it isn't, check the security of your application with [Brakeman](https://github.com/presidentbeef/brakeman)
 * Add bugsnag.
```

Comment:
This should already be added based on https://github.com/infinum/rails-handbook/blob/master/Generating%20New%20Applications.md

Maybe just rename to `Check if bugsnag is included`